x/warp/contract.go

```diff
@@ -195,8 +195,8 @@ func getVerifiedWarpMessage(accessibleState contract.AccessibleState, caller com
 // UnpackSendWarpMessageInput attempts to unpack [input] as []byte
 // assumes that [input] does not include selector (omits first 4 func signature bytes)
 func UnpackSendWarpMessageInput(input []byte) ([]byte, error) {
-	// We don't use strict mode here because it was disabled with the DUpgrade.
-	// Since Warp will be deployed after the DUpgrade, we don't need to use strict mode.
+	// We don't use strict mode here because it was disabled with the Durango.
+	// Since Warp will be deployed after the Durango, we don't need to use strict mode.
```

Comment:
```suggestion
	// We don't use strict mode here because it was disabled with Durango.
	// Since Warp will be deployed after Durango, we don't need to use strict mode.
```